src/lib/format/tests/TestDecoding.cpp

```diff
@@ -274,7 +274,7 @@ void TestFullDataDecoding(nlTestSuite * inSuite, void * inContext)
                    "          endpoint_id: 0\n"
                    "          cluster_id: 49 == 'NetworkCommissioning'\n"
                    "          attribute_id: 65532 == 'featureMap'\n"
-                   "        NetworkCommissioning::featureMap: 4\n"
+                   "        NetworkCommissioning::featureMap: 4 == kEthernetNetworkInterface\n"
```

Comment:
The name of the feature in spec is EthernetNetworkInterface is `EthernetNetworkInterface` or `ET`. Just curious why `kXxxx` format is used.

Reply:
This is how we parse matter files. I think those use `kUpperCamelCase` convention. They can be adjusted/post processed, however this does seem a separate concern.